uber/models.py

```diff
@@ -296,6 +296,9 @@ def _ints(self, name, val):
 
     @suffix_property
     def _label(self, name, val):
+        if not val:
+            return ''
```

Comment:
Well, what this _label property originally did (uh... in some feature branch, at least), was to return the column name if `val` was falsey. I think there's plenty of times that `val` is falsey where it is allowed to be falsey, so I don't think exporting nothing is a problem.

What might be a problem is a few lines down, where it just returns `val` if it can't turn `val` into an integer. We may in that case want to ALSO return an empty string. Hmm... I should probably just go ahead and make that change as part of this PR, actually.